### PR TITLE
IKiselev Task004 done, some changes in Test004

### PR DESCRIPTION
**Pull request by Ikiselev7**
import javafx.concurrent.Task I think we don't need this in test

### CODE REVIEW

File: src/main/java/com/epam/university/java/core/task004/Dictionary.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.epam.university.java.core.task004;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by ilya on 04.09.17.
+ */
+public class Dictionary {
+    private final static Map<String,Integer> DICTIONARY= new HashMap<>();
```
Comment (aabarmin) [inline src/main/java/com/epam/university/java/core/task004/Dictionary.java]: It's better not to hardcode this =)

Comment (Ikiselev7) [reply src/main/java/com/epam/university/java/core/task004/Dictionary.java]: Not to do dictionary or not to do dictionary like this?

Comment (Ikiselev7) [reply src/main/java/com/epam/university/java/core/task004/Dictionary.java]: Not to do dictionary or not to do dictionary like this?

Comment (aabarmin) [reply src/main/java/com/epam/university/java/core/task004/Dictionary.java]: Dictionary like this. Strings here are just for example, here can be another words.

**Pull request closed without merging**